exporter/elasticsearchexporter/bulkindexer.go

```diff
@@ -352,6 +353,9 @@ func flushBulkIndexer(
 		if hint := getErrorHint(resp.Index, resp.Error.Type); hint != "" {
 			fields = append(fields, zap.String("hint", hint))
 		}
+		if resp.Source != "" {
+			fields = append(fields, zap.String("source", resp.Source))
```

Comment:
[to reviewer] this is logged at error level. Should we move this to DEBUG level to avoid sharing sensitive info, or because it is too verbose? But that doesn't sound right to me either if we downgrade the log level.

Possible path forward:
1. Keep logging source field at ERROR level
2. Log source at DEBUG but it will also need the error type and reason etc, which will make it look like a duplicate log.